package.json

```diff
@@ -72,7 +72,7 @@
     "trim": "1.0.1",
     "trim-newlines": "3.0.1",
     "@types/react": "18.0.21",
-    "terser": "5.14.2",
+    "terser": "5.15.1",
```

Comment:
Do we want to update a package that is in `"resolutions"`? cc @sroy3

Reply:
I tested it and everything still works, so it's fine